tket/include/tket/Ops/MetaOp.hpp

```diff
@@ -30,8 +30,6 @@ class MetaOp : public Op {
 
   SymSet free_symbols() const override;
 
-  unsigned n_qubits() const override;
```

Comment:
A bit surprised that `n_qubits()` and `is_equal()` aren't needed but I guess they must not be.

Reply:
Likewise!